src/v1_0/rw/retcode.cpp

```diff
@@ -0,0 +1,35 @@
+#include <abb_librws/v1_0/rw/panel.h>
+#include <abb_librws/v1_0/rws.h>
+#include <abb_librws/rws_retcode.h>
+#include <iostream>
+#include <regex>
+
+namespace abb :: rws :: v1_0 :: rw :: retcode
+{
+    abb::rws::retcode::RetcodeInfo getCode(RWSClient& client, const std::string& code)
+    {
+        std::stringstream uri;
+        uri << Resources::RW_RETCODE << "?code=" << code;
+        POCOResult poco_result = client.httpGet(uri.str());
+        const std::string content = poco_result.content();
+        std::regex re(R"re(<span class=\"(name|code|severity|description)\">(.+?)<\/span>)re");
+        std::smatch match;
+        std::string::const_iterator searchStart(content.cbegin());
+        abb::rws::retcode::RetcodeInfo retcode_info;
+
+        while (regex_search(searchStart, content.cend(), match, re))
+        {
+            if(match[1] == "name")
+                retcode_info.name = match[2];
+            else if (match[1] == "code")
+                retcode_info.code = match[2];
+            else if (match[1] == "severity")
+                retcode_info.severity = match[2];
+            else if (match[1] == "description")
+                retcode_info.description = match[2];
+
+            searchStart = match.suffix().first;
+        }
+        return retcode_info;
```

Comment:
Why using regex instead of an XML parser for XML parsing? `abb_librws` already uses XML parsers from Poco.